internal/app/orchestrator/orchestrator.go

```diff
@@ -385,6 +398,11 @@ func (o *orchestrator) onCacheEvicted(key string, resource cache.Resource) {
 	metrics.OrchestratorCacheEvictSubscope(o.scope, key).Counter(
 		metrics.OrchestratorOnCacheEvictedRequestCount).Inc(int64(len(resource.Requests)))
 	o.logger.With("aggregated_key", key).Debug(context.Background(), "cache eviction called")
+	o.DeleteAll(key, resource)
```

Comment:
was this a bug in current flow?

Reply:
I reverted the change. It is no longer relevant since finding out onCacheEvicted is called for all cache removals.